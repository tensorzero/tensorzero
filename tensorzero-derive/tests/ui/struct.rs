#[derive(tensorzero_derive::TensorZeroDeserialize)]
struct MyStruct {
    field: u8,
}

fn main() {}
