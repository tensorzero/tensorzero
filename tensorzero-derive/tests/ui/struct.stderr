error: TensorZeroDeserialize only supports enums
 --> tests/ui/struct.rs:2:8
  |
2 | struct MyStruct {
  |        ^^^^^^^^
